src/main/java/kr/pickple/back/game/service/GameService.java

```diff
@@ -101,4 +98,27 @@ private GameMember findGameMemberByGameIdAndMemberId(final Long gameId, final Lo
         return gameMemberRepository.findByMemberIdAndGameId(memberId, gameId)
                 .orElseThrow(() -> new GameException(GAME_MEMBER_NOT_FOUND, gameId, memberId));
     }
+
+    @Transactional
+    public void reviewMannerScores(final Long gameId, final List<MannerScoreReview> mannerScoreReviews) {
+        Game game = findGameById(gameId);
+
+        mannerScoreReviews.forEach(review -> {
+            final Member reviewedMember = getReviewedMember(game, review.getMemberId());
+            reviewedMember.updateMannerScore(review.getMannerScore());
+        });
+    }
+
+    private Game findGameById(final Long gameId) {
+        return gameRepository.findById(gameId)
+                .orElseThrow(() -> new GameException(GAME_NOT_FOUND, gameId));
+    }
+
+    private Member getReviewedMember(final Game game, final Long reviewedMemberId) {
+        return game.getMembers(CONFIRMED)
```

Comment:
`p3`

해당 경기에서 확정된 인원만 가져오는 메서드 명인 것으로 보입니다! getMembers를 해올 때 어떤 값을 기준으로 불러온다라는 것이 매개변수를 통해 유추 할 수 있지만 메서드명에서 조금 더 명시되면 좋을 것 같습니다! 아래와 같이 작성하는 것에 대해서는 어떻게 생각하시나요?!

```suggestion
        return game.getMembersByStatus(CONFIRMED)
```

Reply:
반영 : [a68b10c](https://github.com/Java-and-Script/pickple-back/pull/70/commits/a68b10ca29ed9df161098854534086c00e616877)
- 소중한 리뷰 감사합니다 ㅎㅎ